modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using an RPM package. 
```

Comment:
Should we explain which RPM package? Something like "... an RPM package as described in this document"?